inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -116,9 +120,13 @@ static void Transformation(ICNNNetwork::Ptr& clonedNetwork) {
     manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
     manager.register_pass<ngraph::pass::ConvertPrecision>(ngraph::element::i64, ngraph::element::i32);
 
+    OV_ITT_TASK_SKIP(taskChain);
+
     manager.set_callback(transformations_callback);
     manager.run_passes(nGraphFunc);
 
+    OV_ITT_TASK_NEXT(taskChain, "ConvertPrecision");
+
```

Comment:
Do we really need in such detalization of graph pass routine? Do we interested in "RegisterPass", "Prepare" stage?
I suggest to keep only: 
- Apply ngraph common transformations
- Apply ngraph convert precision passes
- convertFunctionToICNNNetwork

Reply:
fixed